packages/fe-fpm-writer/src/page/list.ts

```diff
@@ -14,7 +14,11 @@ function enhanceData(data: ListReport, manifest: Manifest): InternalListReport {
     const config: InternalListReport = { settings: {}, ...data, name: 'ListReport', ...getFclConfig(manifest) };
 
     // move settings into correct possition in the manifest
-    config.settings.entitySet = data.entity;
+    if (data.contextPath) {
+        config.settings.contextPath = data.contextPath;
+    } else {
+        config.settings.entitySet = data.entity;
```

Comment:
@nlunets I need your opinion here. Does it make sense to write either into `contextPath` or `entitySet` or should we consolidate this and always convert an `entitySet` `abc` into `contextPath` `/abc`?

Reply:
@tobiasqueck Contrary to the custom page we here to not have different templates for the different UI5 versions.
Thus we _have_ to provide here two different options, for the different UI5 versions. In older UI5 versions contextPath is not supported.

Reply:
I get that point. My question suggestion was that if someone with e.g. UI5 v1.112 provides the `entity` we still convert it to `contextPath`.

Reply:
@tobiasqueck with the given template for list and object it would be a failure to implicitly assume that `entity` is filled with a contextPath, starting from a certain UI5 release.   Problem is that `entity` is also used as the new target's key, but contextPath definitions normally are very long strings /<entitySet>/ <navigationProperty1>/<navigationProperty2>/...
Thus I would keep both parameters separately. 

What we could do is: _only_  insert contextPath starting from a given UI5 version, for instance 1.94.
This would mean I 
- eliminate options>settings>entitySet from the custom page template for 1.94.
- introduce versioning also for list and object page templates and apply the same here.

Reply:
@tobiasqueck sorry missed that one.

In general it is better to go for contextPath as long as the version supports it, and yes i would even go to the point of replacing when they write entitySet...